changelog.d/pa-1772.fixed

```diff
@@ -0,0 +1,12 @@
+Changed symbolic propagation such that "redundant" matches are no longer reported as findings. For instance:
```

Comment:
I would appreciate hard wrapping very long lines, otherwise it's a bit of a hassle to review them. I don't know if GitHub could be configured to do soft wrapping? But anyways, it's also going to look nicer for those who read the raw changelog.